bindsnet/network/nodes.py

```diff
@@ -324,6 +324,7 @@ def __init__(self, n: Optional[int] = None, shape: Optional[Iterable[int]] = Non
                  trace_tc: Union[float, torch.Tensor] = 5e-2, sum_input: bool = False,
                  thresh: Union[float, torch.Tensor] = -52.0, rest: Union[float, torch.Tensor] = -65.0,
                  reset: Union[float, torch.Tensor] = -65.0, refrac: Union[int, torch.Tensor] = 5,
+                 lbound: Union[float, torch.Tensor] = float('-inf'),
```

Comment:
This is definitely a useful thing to have. Can you add this feature to all other `Nodes` objects with voltages?

Reply:
After some thought, I'm not sure we want this in BindsNET proper. Checking for lower bound at every step in the simulation will be costly. You can totally implement this in your custom `Nodes`, however.

Reply:
How about conditioning the checking operation by 
if self.lbound != float('-inf'):
       self.v.masked_fill_(self.v < self.lbound, self.lbound)
So it won't cost a lot of computation if user doesn't specify the lbound.

Reply:
I think that's reasonable. Maybe default `lbound` to `None`, and check instead `if self.lbound is not None`? Kind of a nitpick, but maybe a little cleaner.